src/store/index.js

```diff
@@ -67,16 +69,16 @@ const store = new Vuex.Store({
   },
 
   getters: {
-    identities: ({ balances }, { addresses }) =>
-      addresses.map(e => ({
+    identities: ({ balances }, { addresses }, {names}) =>
+      addresses.map((e, index) => ({
         balance: balances[e] || 0,
         address: e,
-        name: e.substr(0, 6)
+        name: names[index] || e.substring(0, 6)
```

Comment:
I think we can break compatibility with previous versions using just `names[index]`.